### ISSUE TITLE
Parse link in short source like tweet or toot

### ISSUE THREAD

**Issue by maksim77** [omnivore-app/omnivore#1587]
When I add to read it later service things like tweet or toot that contains link to something probably I later want read content behind this link instead tweet itself.
Maybe we can parse content like this separately and use tweet (toot) only as some kind of metadata?

**Comment by jacksonh** [comment]
yeah, we had an idea for something like this that we've never executed on. Something like having a "Source" so it would save the link, and then also the tweet as the "Source"
